integration-tests/test_184.py

```diff
@@ -1,3 +1,13 @@
+#
+# -*- coding: utf-8 -*-
+#
+# Author: Michael Penkov <m@penkov.dev>
+#
+# Copyright (C) 2019 Radim Rehurek <me@radimrehurek.com>
+#
+# This code is distributed under the terms and conditions
+# from the MIT License (MIT).
+#
```

Comment:
Blank line instead of blank comment, to separate the blocks.

Reply:
I think it's better to keep the entire preamble as a single comment.

If you use blank lines to separate comment blanks, identifying preambles in files (for scripting, etc.) becomes more difficult.

Reply:
It does not separate comment blanks – it separates preamble from code (imports).